pyproject.toml

```diff
@@ -97,3 +97,37 @@ disallow_untyped_defs = true
 testpaths = [
     "tests"
 ]
+
+[tool.hatch.env]
```

Comment:
I haven't heard of hatch before. But in [the docs](https://hatch.pypa.io/latest/why/) they position themselves as replacement for setuptools and tox. So I reckon that, down the line, we could migrate the hatch?